src/Nethermind/Nethermind.Abi.Test/AbiTests.cs

```diff
@@ -367,7 +367,7 @@ public void Test_single_address_no_signature(AbiEncodingStyle encodingStyle)
         [TestCase(AbiEncodingStyle.None)]
         public void Test_packed(AbiEncodingStyle encodingStyle)
         {
-            Keccak assetId = Keccak.Compute("assetId");
+            Keccak assetId = new Keccak(Keccak.Compute("assetId"));
```

Comment:
Why not keep the old signature?

Reply:
Some places don't need the actual `Keccak` object if they then immediately call another method on it like `.ToString`

However reverted the signature and changed those places to use `ValueKeccak.Compute` instead